app/demo-loop/demo-loop.cc

```diff
@@ -131,8 +131,8 @@ void run(std::istream* is, OutputManager* output)
             transport_ptr->params().geometry(),
             transport_ptr->params().action_reg().get());
 
-        // Store input data
-        to_root(root_manager, run_args);
+        // Store input and CoreParams data
+        to_root(root_manager, run_args, transport_ptr->params());
```

Comment:
But doesn't it makes sense to make thee two separate function calls anyway?

Reply:
Initially I had them separate, but later on it seemed more natural to just call `to_root()`, and let this single function take care of storing whatever input / param information needed to fully analyze the step data.

Reply:
I think you should have two functions since there's no interdepdency between them: no data gets passed from one to the next, and it doesn't matter what order you call them.